docs/index-shared.asciidoc

```diff
@@ -0,0 +1,67 @@
+
+:release-state:  unreleased
+:version:        5.5.0
+:major-version:  5.x
+:branch:         5.x
+
+:docker-image:   docker.elastic.co/kibana/kibana:{version}
+:es-ref:         https://www.elastic.co/guide/en/elasticsearch/reference/{branch}/
+:xpack-ref:      https://www.elastic.co/guide/en/x-pack/current/
+:repo:           https://github.com/elastic/kibana/
+:issue:          {repo}issues/
+:pull:           {repo}pull/
+:commit:         {repo}commit/
+:blob:           {repo}blob/{branch}/
+:security:       https://www.elastic.co/community/security/
```

Comment:
Can you increase the spacing between these variables and their contents so that they are all inline with each other?  The spacing should always equal 2 spaces beyond the largest variable name.